packages/nextjs/app/builders/0x224aba5d489675a7bd3ce07786fada466b46fa0f/_components/BuilderHeader.tsx

```diff
@@ -0,0 +1,85 @@
+"use client";
+
+import { BuilderData } from "../types";
+import { SocialLinks } from "./SocialLinks";
+import { getAddress, isAddress } from "viem";
+import { normalize } from "viem/ens";
+import { useEnsAvatar, useEnsName } from "wagmi";
+import { BlockieAvatar } from "~~/components/scaffold-eth";
+import { AddressCopyIcon } from "~~/components/scaffold-eth/Address/AddressCopyIcon";
+import { AddressLinkWrapper } from "~~/components/scaffold-eth/Address/AddressLinkWrapper";
+import { useTargetNetwork } from "~~/hooks/scaffold-eth/useTargetNetwork";
+import { getBlockExplorerAddressLink } from "~~/utils/scaffold-eth";
+
+interface BuilderHeaderProps {
+  builderData: BuilderData;
+}
+
+export const BuilderHeader = ({ builderData }: BuilderHeaderProps) => {
+  const checkSumAddress = (builderData.ethAddress ? getAddress(builderData.ethAddress) : undefined) as `0x${string}`;
+  // const checkSumAddress = "0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1" as `0x${string}`;
```

Comment:
It's bad practice to leave commented code. Other contributors might be confused about whether it should be uncommented or removed.